spec/unit/book_spec.rb

```diff
@@ -16,6 +16,6 @@
   end
   it 'should overwrite parent method if cover_state is bad' do
     book.cover_state = 'bad'
-    expect(book.can_be_archived?).to be 'true'
+    expect(book.send(:can_be_archived?)).to eq true
```

Comment:
The test was failing because I couldn't access a private method. To solve it, I had to implement the send method that you can see in the code above